File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        freq = {}
        n = len(nums)
        k = n // 3
        print(k)
        for n in nums:
            if n in freq:
                freq[n] += 1
            else:
                freq[n] = 1
        print(freq)
        res = []
        for key,val in freq.items():
            if val > k:
                res.append(key)
            else:
                continue
        return res

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    findMin(nums) {

        let res = nums[0]
        for(let i =0; i< nums.length; i++){
             res = Math.min(res, nums[i])
        }
        return res

    }
}
